docs/en-US/component/select-v2.md

```diff
@@ -159,74 +159,80 @@ select-v2/props
+| no-data-text                        | displayed text when there is no options, you can also use slot empty                                                                     | ^[string]                                                                                                                                                                   | No Data       |
+| popper-class                        | custom class name for Select's dropdown                                                                                                  | ^[string]                                                                                                                                                                   | —             |
+| popper-append-to-body ^(deprecated) | whether to append the popper menu to body. If the positioning of the popper is wrong, you can try to set this prop to false              | ^[boolean]                                                                                                                                                                  | false         |
+| teleported                          | whether select dropdown is teleported to the body                                                                                        | ^[boolean]                                                                                                                                                                  | true          |
+| persistent                          | when select dropdown is inactive and `persistent` is `false`, select dropdown will be destroyed                                          | ^[boolean]                                                                                                                                                                  | true          |
+| popper-options                      | [popper.js](https://popper.js.org/docs/v2/) parameters                                                                                   | ^[object]refer to [popper.js](https://popper.js.org/docs/v2/) doc                                                                                                           | {}            |
+| automatic-dropdown                  | for non-filterable Select, this prop decides if the option menu pops up when the input is focused                                        | ^[boolean]                                                                                                                                                                  | false         |
+| height                              | The height of the dropdown panel, 34px for each item                                                                                     | ^[number]                                                                                                                                                                   | 274           |
```

Comment:
Why is it 274? It doesn’t seem to be an integer multiple of 34?

Reply:
Oh, the height of the Select dropdown menu is 274, so I set the same value.

Reply:
Ok.